packages/core/admin/admin/src/content-manager/pages/EditView/components/FormInputs/Component/Input.tsx

```diff
@@ -86,7 +86,7 @@ const ComponentInput = ({
           <NonRepeatableComponent attribute={attribute} name={name} {...props} />
         ) : null}
         {attribute.repeatable && (
-          <RepeatableComponent attribute={attribute} name={name} {...props} />
+          <RepeatableComponent attribute={attribute} name={name} disabled={disabled} {...props} />
```

Comment:
I'll add it for consistency but it doesn't seem to actually be needed, the fields are already disabled. For repeatable components it's needed because there are buttons around the fields (add component, reset component)